src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -6,7 +6,6 @@
 .monaco-workbench > .part.titlebar {
```

Comment:
Getting rid of order makes sense, but all these css changes would be best to tryout and not review imho.